includes/sys_template.php

```diff
@@ -284,7 +284,7 @@ function form_textarea($name, $label, $value, $disabled = false) {
 /**
  * Renders a form selection box
  */
-function form_select($name, $label, $values, $selected) {
+function form_select($name, $label, $values, $selected='') {
```

Comment:
A form may or may not have a selected field value. That case is handled. Helps in removing an error in guest_login_controller.php